pyvista/plotting/colors.py

```diff
@@ -174,7 +184,6 @@
 hexcolors = {
```

Comment:
Note that the `hexcolors` dict, as the comment above it, says that it was copied from matplotlib. This might explain why paraview's background was not inserted in there. We should at least amend the comment mentioning some amendments (removing duplicates and adding paraview, if we keep this state).